src/test/java/cz/czechitas/TemplateTest.java

```diff
@@ -1,24 +1,57 @@
 package cz.czechitas;
 
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.*;
+
+import static org.junit.jupiter.api.Assertions.*;
 
 public class TemplateTest {
 
+    private static WebDriver driver;
+    private static AdminPage adminPage;
 
     @BeforeEach
-    public void before() {
-
+    public void before() throws InterruptedException {
+        driver = DriverHelper.openFirefox();
+        adminPage = new AdminPage(driver);
+        adminPage.openPage();
     }
 
     @AfterEach
     public void after() {
+        driver.quit();
+    }
 
+    @Test
+    @DisplayName("Login with invalid password")
+    public void invalidPasswordTest() throws InterruptedException {
+        adminPage.login("admin@shopizer.com", "password1");
+        Thread.sleep(5000L);
+        String expectedErrorMessage = "Invalid username or password";
+        WebElement errorMessage = By.xpath("//div[@class='alert alert-error']").findElement(driver);
+        String actualErrorMessage = errorMessage.getText();
+        assertEquals(actualErrorMessage, expectedErrorMessage, "Login enabled with wrong password");
+    }
+
+    @Test
+    @DisplayName("Login with invalid username")
+    public void invalidUsernameTest() throws InterruptedException {
+        adminPage.login("admin@shopizer1.com", "password");
+        Thread.sleep(5000L);
+        String expectedErrorMessage = "Invalid username or password";
```

Comment:
Stejnou promenou mas uz v predeslem testu, asi by se to tedy dalo "vytahnout" na uroven tirdy